deployment/localsettings.template.py

```diff
@@ -1,11 +1,11 @@
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql',   # postgresql', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': 'evap',                         # Or path to database file if using sqlite3.
-        'USER': 'evap',                         # Not used with sqlite3.
-        'PASSWORD': 'evap',                     # Not used with sqlite3.
-        'HOST': '127.0.0.1',                    # Set to empty string for localhost. Not used with sqlite3.
-        'PORT': '',                             # Set to empty string for default. Not used with sqlite3.
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'evap',                       
+        'USER': 'evap',                     
+        'PASSWORD': 'evap',             
```

Comment:
```suggestion
        'ENGINE': 'django.db.backends.postgresql',
        'NAME': 'evap',
        'USER': 'evap',
        'PASSWORD': 'evap',
```

(We don't have auto-formatting for this file, I still don't want trailing spaces in here)